### DISCUSSION TITLE
I am doing spacy custom NER with my own data. and i train the model using spacy CLI command and predicting using model load

### DISCUSSION THREAD

**Discussion by hiteshpara** [explosion/spaCy#9892] [Help: Coding & Implementations]
## Your Environment

* Operating System:
* Python Version Used:
* spaCy Version Used:
* Environment Information:

**Comment by svlandeg** [comment]
Hi, I've moved your thread from the issue tracker to the discussions forum here.

I'm not sure what your question is though?
